.github/workflows/publish-package.yml

```diff
@@ -40,5 +34,36 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WDIO_TOKEN }}
+
+      - name: Install packages
+        run: yarn install
+
       - name: Release new package
-        run: standard-version --dry-run
+        run: |
+          git config user.name ${{ secrets.USERNAME }}
+          git config user.email "${{ secrets.MAIL }}"
+          yarn release
```

Comment:
need a better name for this phase. it's not releasing a package at this point

Reply:
done